pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
```

Comment:
> Not sure if it would be better to list the files instead of copying them, as a backend like flit could satisfy the current method with a shutil.copytree().

If the hook is specified as defining the minimal set of files needed to build a wheel, flit will probably do something slightly more advanced than calling `copytree()` on the whole directory, for performance reasons. I think the implementation difficulty would be similar whether it returns a list of files or copies them itself.

If the hook actually generates something like an sdist, as Donald favours, then it's a much more complex question.